tfs/src/main/java/hudson/plugins/tfs/model/GitCodePushedEventArgs.java

```diff
@@ -2,10 +2,11 @@
 
 import org.eclipse.jgit.transport.URIish;
 
+import java.io.Serializable;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-public class GitCodePushedEventArgs {
+public class GitCodePushedEventArgs implements Serializable {

```

Comment:
Well you have to consider whether this is a plausible situation where someone might have subclassed it to begin with.